CMakeLists.txt

```diff
@@ -675,6 +675,18 @@ add_dependencies(lintcommit nvim)
 add_custom_target(lint)
 add_dependencies(lint check-single-includes lintc lintlua lintpy lintsh lintcommit lintuncrustify)
 
+#
+# Format
+#
+
```

Comment:
with a blank line, these kinds of headers tend to get lost because the connection isn't obvious. I suggest smushing the related parts together without blank lines.